### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
image

### CODE REVIEW

File: src/components/feed/post-minified.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const MinifiedPost = ({ post }: { post: MinifiedPostType }) => {
+  return (
+      <li className="hover:bg-light-primary-0.08-tlc">
+        <button className="card container-surface text-left">
+          <Title
+            icon={
+              <div className="grid place-items-center w-10 h-10
+                text-light-on-secondary font-medium bg-light-secondary rounded-full">
+                {post.author[0]}
```
Comment (smartwhatt) [inline src/components/feed/post-minified.tsx]: Can you trust that the backend will send back a username with at least 1 letter?

File: src/components/feed/post.tsx [hunk 2]
```diff
@@ -21,7 +21,8 @@ const Post = ({ post }: { post: PostType }) => {
     <li className="w-full max-w-full sm:card sm:card-elevated sm:w-[74ch]">
       <Title
         icon={
-          <div className="grid place-items-center w-10 h-10 text-light-on-secondary font-medium bg-light-secondary rounded-full">
+          <div className="grid place-items-center w-10 h-10
+            text-light-on-secondary font-medium bg-light-secondary rounded-full">
             {post.author[0]}
```
Comment (smartwhatt) [inline src/components/feed/post.tsx]: same here

File: src/components/schedule/index.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                        dark:text-dark-on-surface dark:bg-dark-surface1 transition-shadow overflow-hidden
+                          hover:shadow focus-visible:shadow"
+                      >
+                        <p className="text-lg font-bold max-lines-1">
+                          {scheduleItem.periodLength > 75
+                            ? scheduleItem.subject.enName.name
+                            : scheduleItem.subject.enName.shortName}
+                        </p>
+                        <p className="text-base max-lines-1">
+                          {scheduleItem.subject.instructors[0]?.firstName}
```
Comment (smartwhatt) [inline src/components/schedule/index.tsx:57]: What if the backend doesn't send any instructors or the code is not working correctly?
what should be shown to the user?

File: src/pages/index.tsx [hunk 4]
```diff
@@ -38,6 +33,14 @@ const Dashboard: NextPage = () => {
                 urgent: false,
               },
             ],
+            subject: {
+              
+            name: "English 8",
+            instructors: [
+              { firstName: "John Peter", lastName: "Smith" },
```
Comment (smartwhatt) [inline src/pages/index.tsx:40]: MIDDLE NAME

File: src/styles/global.css [hunk 5]
```diff
@@ -77,6 +77,10 @@
   .card-elevated {
     @apply container-surface shadow;
   }
+
+  .card-tonal {
```
Comment (smartwhatt) [inline src/styles/global.css:86]: Why do we need this?
There's only 1 class applied

File: src/styles/global.css [hunk 6]
```diff
@@ -100,6 +104,11 @@
       dark:text-dark-on-surface dark:bg-dark-surface;
   }
 
+  .container-surface-variant {
```
Comment (smartwhatt) [inline src/styles/global.css:112]: Here could be 
```css
.container-surface-variant, .card-tonal {
    ...
}
```

**Review by smartwhatt — APPROVED** [review]
A few things to fix.
Merge when done

File: src/components/schedule/index.tsx (hunk 3 above)
Comment (SiravitPhokeed) [reply src/components/schedule/index.tsx:57]: image
And to be honest, I'm fine with this.

File: src/styles/global.css (hunk 6 above)
Comment (SiravitPhokeed) [reply src/styles/global.css:112]: I can't do that because `card-tonal` is a component, not a utility.

**Review by ImSadudee — APPROVED** [review]
Pogger pogress pog pog

**Pull request merged**